### PR TITLE
Add changes for 10th task.

### PR DESCRIPTION
**Pull request by mightveseen**

### CODE REVIEW

File: delivery/meteor-delivery.js [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 
-router.get('/api/meteors', async (req, res) => {
+router.get('/meteors', async (req, res, next) => {
     try {
         const params = {
             date: req.query.date || new Date().toISOString().split('T')[0],
-            count: req.query.count || -1,
-            isDangerous: req.query['were-dangerous-meteors'] === 'true' || undefined
+            count: String(req.query.count).toLowerCase() === 'true',
+            isDangerous: req.query['were-dangerous-meteors']
```
Comment (mdziarkach) [inline delivery/meteor-delivery.js]: all types should be prepared for use cases inside of controller
isDangerous - is boolean? then need to provide correct type

File: use_cases/meteor-use-cases.js [hunk 2]
```diff
@@ -12,12 +11,15 @@ export function getMeteors(data, params) {
...
+        count: params.count ? result.length : undefined,
+        meteors: result
+    };
 }
 
 function flatAndFilterArray(data, params) {
     const meteors = Object.values(data['near_earth_objects']).flat();
     return params.isDangerous !== undefined
-        ? meteors.filter(meteor => meteor['is_potentially_hazardous_asteroid'] === params.isDangerous)
+        ? meteors.filter(meteor => meteor['is_potentially_hazardous_asteroid'] === (params.isDangerous.toLowerCase() === 'true'))
```
Comment (mdziarkach) [inline use_cases/meteor-use-cases.js]: all data should be with correct type on this layer

File: app.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { config } from 'dotenv';
+import nunjucks from 'nunjucks';
+import express from 'express';
+import meteorDelivery from './delivery/meteor-delivery.js';
+
+const app = express();
+const port = process.env.SERVER_PORT || 4000;
```
Comment (mdziarkach) [inline app.js]: add config.js file for managing all env variables

File: delivery/meteor-delivery.js (hunk 1 above)
Comment (mightveseen) [reply delivery/meteor-delivery.js]: Fixed.

File: use_cases/meteor-use-cases.js (hunk 2 above)
Comment (mightveseen) [reply use_cases/meteor-use-cases.js]: Fixed.

File: app.js (hunk 3 above)
Comment (mightveseen) [reply app.js]: Fixed.

**Review by mdziarkach — APPROVED** [review]

**Pull request merged**